actnow/profiles/signals.py

```diff
@@ -14,3 +18,15 @@ def number_of_organisations_changed(sender, **kwargs):
     if kwargs["action"] == "post_remove":
         if sender.objects.count() < 1:
             raise ValidationError("An Organisation must have at least one person.")
+
+
+@receiver(post_save, sender=User)
+def create_profile(sender, instance=None, created=False, **kwargs):
+    if created:
+        data = instance.extra_fields["request_data"]
```

Comment:
```suggestion
        data = instance.extra_fields.get("request_data")
```
I think `request_data` key will be there when run from management command e.g. `createsuperuser`